documentation/asciidoc/computers/configuration/headless.adoc

```diff
@@ -1,39 +1,41 @@
-
-=== Configuring Networking
+=== Connect to a network
 
 To configure network access at first boot in a headless Raspberry Pi, use the advanced settings menu in Rasbperry Pi Imager. Enter the SSID and password of your preferred wireless network. Your Raspberry Pi will use these credentials to connect to the network on first boot. Some wireless adapters and some Raspberry Pi boards do not support 5Ghz networks; check the documentation for your wireless module to ensure compatibility with your preferred network.
 
-With no keyboard or monitor, you need some way of xref:remote-access.adoc[remotely accessing] your headless Raspberry Pi. To enable SSH:
+WARNING: Depending on the model and SD card, your Raspberry Pi may require up to 5 minutes to boot and connect to your wireless network.
+
+=== Remote access
+
+With no keyboard or monitor, you need a way to xref:remote-access.adoc[remotely control] your headless Raspberry Pi. On first boot, the only option is SSH. To enable SSH on a fresh installation of Raspberry Pi OS, choose one of the following methods:
 
 * enable SSH in the Advanced Menu of Raspberry Pi Imager, then enter a username and password
-* place a file named `ssh` (no file extension) into the boot folder (`/boot/firmware/`) of the SD Card, then provide a username and password xref:headless.adoc#configuring-a-user[via `userconf.txt`].
+* place a file named `ssh` (no file extension) into the boot folder (`/boot/firmware/`) of the SD Card, then configure a user with `userconf.txt`
 
-For more information, see xref:remote-access.adoc#ssh[set up an SSH server].
+IMPORTANT: To use SSH on a fresh Raspberry Pi OS installation, you must create a user account in the xref:getting-started.adoc#advanced-options[Advanced Menu] of Raspberry Pi Imager.
 
-WARNING: Depending on the model and SD card, your Raspberry Pi may require up to 5 minutes to boot and connect to your wireless network.
+For more information, see xref:remote-access.adoc#ssh[set up an SSH server]. Once you've connected over SSH, you can use `raspi-config` to xref:remote-access.adoc#vnc[enable VNC] if you'd prefer a graphical desktop environment.
 
 [[configuring-a-user]]
-=== Configuring a User
+=== Configure a User
 
-You will need to add a `userconf.txt` in the boot partition of the SD card; this is the part of the SD card which can be seen when it is mounted in a Windows or MacOS computer.
+In the boot partition of your SD card (`/boot/firmware`), create a file named `userconf.txt`.
```

Comment:
In contrast to other places where the boot partition is mentioned, I think you _shouldn't_ mention `/boot/firmware` here; because that's only the path that the boot partition appears at once you're actually booted into Raspberry Pi OS, but these instructions are about editing files (from another computer) _before_ you boot Raspberry Pi OS for the first time.

Reply:
Aha, excellent point. I now see why the original sentence included the confusing bit about the "part which is seen when it is mounted in a [separate] computer". Fixing now.